### DISCUSSION TITLE
Reusing Spectral Libraries in DiaNN Analysis

### DISCUSSION THREAD

**Discussion by enriiquee** [vdemichev/DiaNN#1012] [Q&A]
Hello everyone,

I'm new to using DiaNN and currently facing an issue with data analysis. After generating a spectral library during the initial pass, I proceed to analyze my data. However, during the second round of analysis using the generated library, the process occasionally halts without any error messages. Subsequently, when I attempt to rerun the analysis, DiaNN regenerates the spectral library from scratch. Is there a method to instruct the program to utilize the previously generated library from the initial pass?

Thank you for your assistance.

**Comment by vdemichev** [comment]
Hi Enrique,

The first pass library is saved to the 'Output library' location, and this should also be indicated in the log. By default, the output library name will end with '-lib.tsv', i.e. can see if it's in the output folder.

The fact that it crashes seems unusual, is it's some beta version? (1.8.1 is known to be stable)

Best,
Vadim
